content/20.zksync-protocol/00.rollup/10.transaction-lifecycle.md

```diff
@@ -100,14 +100,12 @@ adds an `accessList` to transactions, which is an array of addresses and storage
 
 ### EIP-1559: `0x2`
 
-Introduced in Ethereum's London update, [EIP-1559: Fee market change for ETH 1.0 chain](https://eips.ethereum.org/EIPS/eip-1559)
-modifies how transaction fees are handled, replacing `gasPrice` with a base fee and allowing users to set `maxPriorityFeePerGas` and `maxFeePerGas`.
+Ethereum's [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) replaces `gasPrice` with a base fee and adds parameters like `maxFeePerGas`.
 
-- `maxPriorityFeePerGas`: Maximum fee users are willing to pay miners as an incentive.
```

Comment:
I think it still makes sense to mention it with something like "Determines the priority fee. Since ZKsync has no concept of priority fee, should be set to 0"